Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -32,7 +32,7 @@ @implementation _CTFramesetter : NSObject
     // Call _DWriteWrapper to get _CTLine object list that makes up this frame
     _CTTypesetter* typesetter = static_cast<_CTTypesetter*>(framesetter->_typesetter);
     if (range.length == 0L) {
-        range.length = typesetter->_characters.size();
+        range.length = typesetter->_characters.size() - range.location;
     }
```

Comment:
like: do they accrue to another bugfix?

---
In reply to: [87268996](https://github.com/Microsoft/WinObjC/pull/1343#discussion_r87268996) [](ancestors = 87268996)